kano_profile/badges.py

```diff
@@ -69,6 +69,30 @@ def calculate_xp():
     return int(points) + qm.evaluate_xp()
 
 
+def get_group_progress(group_profile):
+    if 'challengeNo' not in group_profile:
```

Comment:
:+1:
We probably need to reorganise these fns into existing and new files. The existing structure is very unclear.